e2e/preferences.e2e.spec.ts

```diff
@@ -0,0 +1,144 @@
+import os from 'os';
+import path from 'path';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import { PreferencesPage } from './pages/preferences';
+
+let page: Page;
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('Main App Test', () => {
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+  let preferencesWindow: Page;
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        // See src/utils/commandLine.ts before changing the next item as the final option.
+        '--disable-dev-shm-usage',
+        '--no-modal-dialogs',
+      ],
+      env: {
+        ...process.env,
+        RD_LOGS_DIR: reportAsset(__filename, 'log'),
+      },
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+
+    new NavPage(page).preferencesButton.click();
+
+    await electronApp.waitForEvent('window');
+    await electronApp.waitForEvent('window');
+
+    const windows = electronApp.windows();
+
+    preferencesWindow = windows.find(w => w.url().toLocaleLowerCase().includes('preferences')) || await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: reportAsset(__filename) });
+    await packageLogs(__filename);
+    await electronApp.close();
+  });
+
+  test('should open preferences modal', () => {
+    expect(preferencesWindow).toBeDefined();
+    expect(preferencesWindow?.url().toLowerCase()).toContain('preferences');
+  });
+
+  test('should land application behavior tab', async() => {
```

Comment:
```suggestion
  test('should show application behavior tab', async() => {
```
I'm not quite sure about this, other than `… should land … tab` sounds odd.